open-media-match/src/OpenMediaMatch/__init__.py

```diff
@@ -1,13 +1,15 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
 
+import os
 import flask
 
 
 def create_app():
     """
     Create and configure the Flask app
     """
-    app = flask.Flask(__name__, instance_relative_config=True)
+    app = flask.Flask(__name__)
+    app.config.from_envvar('OMM_CONFIG')
```

Comment:
(ignorable): I think having this be a harcoded path at this stage is also fine, one less indirection to follow